### PR TITLE
feat(kmp): improve docs

### PR DESCRIPTION
**Pull request by buenaflor**
This is a continuation of #6975 and since I'm currently not a part of the sentry org anymore, I have to pull in from a fork and cannot push onto the original pr branch.

@shanamatthews  @marandaneto

### CODE REVIEW

**Review by marandaneto — APPROVED** [review]
Thanks @buenaflor

**Review by cleptric — APPROVED** [review]

File: src/platforms/kotlin-multiplatform/common/install/swift-package-manager.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Add the Sentry Cocoa SDK as a package in Xcode in your iOS app via **File > Add Packages**.
+Enter the Git repo URL in the search field:
+
+```text
+https://github.com/getsentry/sentry-cocoa.git
+```
+
+Define your dependency rule to have the exact version `8.4.0` and then click the "Add Package" button.
+
+Alternatively, when your project uses a `Package.swift` file to manage dependencies, you can specify the target with:
```
Comment (shanamatthews) [inline src/platforms/kotlin-multiplatform/common/install/swift-package-manager.mdx:20]:
```suggestion
Alternatively, if your project uses a `Package.swift` file to manage dependencies, you can specify the target with:
```

**Review by shanamatthews — APPROVED** [review]
This all looks good to me. I made some more suggestions (and added a change in `/getting-started-primer/kotlin-multiplatform.mdx`, which wasn't in this PR) to further clarify the SDK version note.

**Comment by buenaflor** [comment]
Thx for the corrections Shana, it's ready to be merged from my side!

**Comment by marandaneto** [comment]
@buenaflor can you fix the lint issue? then we are able to merge it.

> lint produced file changes, run linter locally and try again' && exit 1

**Comment by buenaflor** [comment]
@marandaneto should be fixed now

**Pull request merged**